frontstage/views/account/account_transfer_survey.py

```diff
@@ -92,11 +93,71 @@ def transfer_survey_email_entry(session):
             return render_template("surveys/surveys-transfer/recipient-email-address.html", form=form, errors=errors)
 
         flask_session["transfer_survey_recipient_email_address"] = form.data["email_address"]
+
+        existing_pending_surveys = get_existing_pending_surveys(party_id)
+
+        if existing_pending_surveys:
+            duplicate_transfers = _get_duplicate_transfers(existing_pending_surveys, form.data["email_address"])
+            if duplicate_transfers:
+                business_survey_enrolments = get_business_survey_enrolments_map(party_id)
+                errors = {
+                    "email_address": [
+                        _build_duplicate_transfer_error_message(duplicate_transfers, business_survey_enrolments)
+                    ]
+                }
+                return render_template(
+                    "surveys/surveys-transfer/recipient-email-address.html", form=form, errors=errors
+                )
+
         return redirect(url_for("account_bp.send_transfer_instruction_get"))
 
     return render_template("surveys/surveys-transfer/recipient-email-address.html", form=form)
 
 
+def _get_duplicate_transfers(existing_pending_surveys, email_address):
+    duplicate_transfers = []
+    for existing_pending_survey in existing_pending_surveys:
+        if (
+            _pending_survey_for_business_exists(
+                flask_session["surveys_to_transfer_map"],
+                existing_pending_survey["business_id"],
+                existing_pending_survey["survey_id"],
+            )
+            and existing_pending_survey["email_address"].lower() == email_address.lower()
+        ):
+            duplicate_transfers.append(
+                {
+                    "business_id": existing_pending_survey["business_id"],
+                    "survey_id": existing_pending_survey["survey_id"],
+                    "email_address": existing_pending_survey["email_address"].lower(),
+                }
+            )
+    return duplicate_transfers
+
+
+def _build_duplicate_transfer_error_message(duplicate_transfers, business_survey_enrolments):
```

Comment:
This can probably be improved as it's clunky chaining this and including the HTML. But it's explicit and serves its purpose.